std_msgs/Header header
uint8 quantity_of_params
uint8[] numbers
float32[] param_values
